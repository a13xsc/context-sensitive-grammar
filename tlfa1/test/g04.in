({R,P,J,N,O,U,_,D},{_},{},J)
