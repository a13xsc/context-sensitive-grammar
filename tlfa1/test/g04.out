U
N
D
J
O
P
R
